app/views/assignments/_cp_assignment_download_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= content_for :modal_id, 'download_dialog' %>
```

Comment:
We're not going to be offering any options here for how to download the file (in other places we offer different file formats but here it's just going to be a zip file). So there is no need to have a modal for this. When you click on the download link it should just download the file without any intermediate steps.